### ISSUE TITLE
Defining color values should be more liberate

### ISSUE THREAD

**Issue by ChristophLabacher** [opencolor-tools/opencolor-js#20]
```
greyDarkestTransparent: rgba(0,0,0,.34)
```

Expected result: Transparent Color Swatch
Actual result: Parser Error (`Unexpected '.'`)

This works fine:
```
greyDarkestTransparent: rgba(0,0,0,0.34)
```

---
```
hsl(20,60,50)
```

Expected result: Brown Color Swatch
Actual result: White Color Swatch

This works fine:
```
hsl(20,60%,50%)
```

**Comment by halfbyte** [comment]
First one is legit, that should probably be supported. 

Second one is a bit trickier: The issue is that we currently do not parse colors in the parser at all (with the exception of hexcolors and simply assume a format of foo(bar) where bar can be a list of comma separated things and foo is an arbitrary color space. So I would assume some sort of problem in the current swatch rendering code. What do you say, @theflow?

**Comment by halfbyte** [comment]
Technically, on the parser level, I've tested all of your inputs here and they parse fine. 

Interpreting them correctly is a different question, but I'm closing this issue for the same reason as I closed #21, the Trello ticket for this is https://trello.com/c/tvUJfcaI
